libs/utils/src/namespaces/tests/fonts.test.ts

```
@@ -0,0 +1,24 @@
+import { describe, it, expect } from "vitest";
+import { isLocalFont, localFonts } from "../fonts";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)